src/merge.js

```diff
@@ -0,0 +1 @@
+console.log('Hello Yay');
```

Comment:
hey maybe your forgot to remove this line or this file? It is not clear what this file is for... thanks.